ReactWindows/ReactNative.Net46/Views/Image/ReactImageManager.cs

```diff
@@ -306,7 +306,7 @@ private void OnImageStatusUpdate(Border view, ImageStatusEventData status)
                     status => OnImageStatusUpdate(view, status),
                     _ => OnImageFailed(view));
 
-                image.UriSource = new Uri(source);
+                image.UriSource = new Uri(source, UriKind.Absolute);
```

Comment:
>UriKind [](start = 50, length = 7)

Do you need to qualify this Absolute here?  Doesn't the Uri constructor handle this by default? [https://referencesource.microsoft.com/#System/net/System/URI.cs,399](https://referencesource.microsoft.com/#System/net/System/URI.cs,399)